target/linux/ipq40xx/image/generic.mk

```diff
@@ -711,6 +711,19 @@ define Device/linksys_mr8300
 endef
 TARGET_DEVICES += linksys_mr8300
 
+define Device/linksys_whw03
+	$(call Device/FitzImage)
+	DEVICE_VENDOR := Linksys
+	DEVICE_MODEL := WHW03
+	SOC := qcom-ipq4019
+	KERNEL_SIZE := 8192k
+	IMAGE_SIZE := 131072k
+	IMAGES += factory.bin
+	IMAGE/factory.bin  := append-kernel | pad-to $$$$(KERNEL_SIZE) | append-rootfs | pad-rootfs | linksys-image type=WHW03
+	DEVICE_PACKAGES := uboot-envtools ath10k-firmware-qca9888-ct ipq-wifi-linksys_whw03 kmod-leds-pca963x kmod-fs-ext4 kmod-fs-f2fs mkf2fs losetup
```

Comment:
Why you need kmod-fs-ext4 kmod-fs-f2fs mkf2fs losetup in default image?

Reply:
kmod-fs-ext4 is because V1 uses eMMC rather than flash and the thinking was that ext4 was the more appropriate type - (V2 does not use eMMC). As for f2fs, mkf2fs and losetup I think they were for debugging/testing. Let me verify with a test build and then I can add a commit to remove what is not needed

Reply:
@obsy - I've removed those packages, tested a new build, looks good